analysis/tc_daytrader_deps.go

```diff
@@ -19,71 +19,41 @@ var DaytraderWithDeps = TC{
 		Issues: []api.Issue{
 			{
 				Category:    "mandatory",
-				Effort:      1,
 				Description: "File system - Java IO",
+				Effort:      1,
 				RuleSet:     "cloud-readiness",
 				Rule:        "local-storage-00001",
 				Incidents: []api.Incident{
 					{
-						File: "/shared/source/sample/daytrader-ee7-web/src/main/java/com/ibm/websphere/samples/daytrader/web/TradeScenarioServlet.java",
-						Line: 125,
-						Message: `An application running inside a container could lose access to a file in local storage.
-
- Recommendations
-
- The following recommendations depend on the function of the file in local storage:
-
- * Logging: Log to standard output and use a centralized log collector to analyze the logs.
- * Caching: Use a cache backing service.
- * Configuration: Store configuration settings in environment variables so that they can be updated without code changes.
- * Data storage: Use a database backing service for relational data or use a persistent data storage system.
- * Temporary data storage: Use the file system of a running container as a brief, single-transaction cache.`,
+						File:    "/shared/source/sample/daytrader-ee7-web/src/main/java/com/ibm/websphere/samples/daytrader/web/TradeScenarioServlet.java",
+						Line:    125,
+						Message: "An application running inside a container could lose access to a file in local storage.",
```

Comment:
Just little cleanup for for actual results, expecting its update in https://github.com/konveyor/go-konveyor-tests/issues/277